operators/cp4ba-operator/base/subscription.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: ibm-cp4a-operator-catalog-subscription
+  namespace: cp4ba
+spec:
+  channel: v22.1
```

Comment:
Since the Customisation file in overlays/latest already changes this to the required value, I would change the value to something like replaceme to make it obvious to the user that this is a configurable parameter.